source/_components/light.mysensors.markdown

```diff
@@ -108,6 +108,146 @@ void incomingMessage(const MyMessage &message) {
     }
 }
 ```
+### {% linkable_title MySensors 2.x example sketch %}
 
+```cpp
+/**
+ * Created by Henrik Ekblad <henrik.ekblad@mysensors.org>
+ * Copyright (C) 2013-2015 Sensnology AB
+ * Full contributor list: https://github.com/mysensors/Arduino/graphs/contributors
```

Comment:
I prefer if you just link to the original sketch at the mysensors build site instead of having all this attribution info.